go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go

```diff
@@ -115,75 +115,29 @@ func TestSingleQuery(t *testing.T) {
 
 // Convenience test for debugging a single query. Unskip and set to the desired query.
 func TestSingleScript(t *testing.T) {
-	t.Skip()
+	//t.Skip()
 
 	var scripts = []queries.ScriptTest{
 		{
-			Name: "datetime precision",
+			// panic: runtime error: slice bounds out of range [-1:]
+			Name: "Slice out of bounds panic repro",
```

Comment:
Why no new engine test that would catch this?

Reply:
Sorry, I should have highlighted that better – I added tests to the PrefixIndex tests in GMS PR linked from the description. Those tests do trigger the same collation error that happened in the linked issue. 
https://github.com/dolthub/go-mysql-server/pull/1971